### PR TITLE
Add the extensions back in

### PR DESCRIPTION
**Pull request by eyedeekay**
I forgot to 'git add' the extensions folder. Missed it because I installed with 'make install' and binaries are just the one line in the code review. This fixes it.

### CODE REVIEW

**Comment by adrelanos** [comment]
We've never added binary files to Whonix source code before. Issues:

* need to verify the files being non-malicious
* maintenance intense - needs to be updated every time Tor Project releases new versions
* we'd introduce a blocker level Debian policy violation

Could you implement this some other way please? Such as copying these files from the extracted Tor Browser archive?

//cc @HulaHoopWhonix (To make sure I am not requesting something unreasonable here.)

**Comment by eyedeekay** [comment]
Not a problem at all. I've removed them from the repo and instead I'm copying them from profile.default in the ~/.i2pb/i2p-browser directory only after were sure update-i2pbrowser has been run.

File: debian/copyright [hunk 1]
```diff
@@ -51,3 +51,7 @@ License: BSD-3-clause
  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Files: usr/share/tb-profile-i2p/profile.i2p/extensions/{73a6fe31-595d-460b-a920-fcc0f8843232}.xpi
+Copyright: Copyright 2004-2018 Giorgio Maone <g.maone@informaction.com>
+License: GPL-2+
```
Comment (adrelanos) [inline debian/copyright]: Looks obsolete now.

File: usr/bin/i2pbrowser [hunk 2]
```diff
@@ -17,6 +17,11 @@ fi
 cp -r --no-clobber /usr/share/tb-profile-i2p/profile.i2p \
     "$HOME/.i2pb/profile.i2p"
 
+rm -rf "$HOME/.i2pb/profile.i2p/extensions"
+
+cp -r --no-clobber "$HOME/.i2pb/i2p-browser/Browser/TorBrowser/Data/Browser/profile.default/extensions" \
+    "$HOME/.i2pb/profile.i2p/extensions"
+
```
Comment (adrelanos) [inline usr/bin/i2pbrowser]: Looks ok.

**Comment by eyedeekay** [comment]
I mangled the history. I'll be right back.

**Pull request closed without merging**